Model/Magento/CatalogImportExport/Import/Product/Product.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace FireGento\ExtendedImport\Model\Magento\CatalogImportExport\Import\Product;
+
+use Magento\CatalogImportExport\Model\Import\Product\RowValidatorInterface as ValidatorInterface;
+use Magento\ImportExport\Model\Import\ErrorProcessing\ProcessingError;
+use FireGento\ExtendedImport\Model\Magento\CatalogImportExport\ResourceModel\Import\Data;
+
+/**
+ * Class Product
+ * @package FireGento\ExtendedImport\Model\Magento\CatalogImportExport\Import\Product
+ */
+class Product extends \Magento\CatalogImportExport\Model\Import\Product
+{
+    /**
+     * Add error with corresponding current data source row number.
+     *
+     * @param string $errorCode Error code or simply column name
+     * @param int $errorRowNum Row number.
+     * @param string $colName OPTIONAL Column name.
+     * @param string $errorMessage OPTIONAL Column name.
+     * @param string $errorLevel
+     * @param string $errorDescription
+     * @return $this
+     */
+    public function addRowError(
+        $errorCode,
+        $errorRowNum,
+        $colName = null,
+        $errorMessage = null,
+        $errorLevel = ProcessingError::ERROR_LEVEL_CRITICAL,
+        $errorDescription = null
+    ) {
+        $errorCode = (string)$errorCode;
+
+        if ($errorCode !== ValidatorInterface::ERROR_DUPLICATE_URL_KEY) {
```

Comment:
is this really needed? cause the error is fixed in generating URLs ... so we should not suppress it?

Reply:
@amenk Actually yes... The validation happens before saving batches to raw, and overriding to have this disable the rule in the core is just too hard. So the solution was just to disable this kind of exception.

Reply:
ok